isb_web/export.py

```diff
@@ -174,7 +174,7 @@ async def create_sitemap(request: fastapi.Request, session: Session = Depends(ge
     orcid_id = auth.orcid_id_from_session_or_scope(request)
     if orcid_id not in isb_web.config.Settings().orcid_superusers:
         raise HTTPException(401, "orcid id not authorized to create sitemap")
-    solr_api_defparams = _default_export_params("indexUpdatedTime asc")
+    solr_api_defparams = _default_export_params("sourceUpdatedTime asc")
```

Comment:
This change is non-trivial.  The implication is that if we were to, say, reindex a bunch of stuff on the iSB instance, we wouldn't get it by default, because the `sourceUpdatedTime` wouldn't be changed.  In a world where we don't own the boxes, this would be up to the provider to determine, so I think it's the right thing to do in the meantime, just FYI.